main.js

```diff
@@ -241,87 +284,45 @@ function createPopup(index) {
   });
 }
 
-for (let i = 0; i < projectsArray.length; i += 1) {
-  const projectWrapper = document.getElementsByClassName('project_wrapper')[0];
-
-  const projectCard = document.createElement('article');
-  projectCard.classList.add('project_card', `card${i}`);
-
-  const projectCardImage = document.createElement('img');
-  projectCardImage.classList.add('project_image');
-  projectCardImage.setAttribute('src', projectsArray[i].imgMobile);
-  projectCardImage.setAttribute('alt', projectsArray[i].name);
-  projectCard.appendChild(projectCardImage);
-
-  const projectCardWrapper = document.createElement('div');
-  projectCardWrapper.classList.add('project_card_wrapper');
-  projectCard.appendChild(projectCardWrapper);
-
-  const projectCardTitle = document.createElement('h4');
-  projectCardTitle.classList.add('project_card_title');
-  projectCardTitle.innerText = projectsArray[i].name;
-  projectCardWrapper.appendChild(projectCardTitle);
-
-  const projectCardTags = document.createElement('ul');
-  projectCardTags.classList.add('project_card_tags');
-  projectCardWrapper.appendChild(projectCardTags);
-
-  for (let j = 0; j < projectsArray[i].techtags.length; j += 1) {
-    const projectCardTag = document.createElement('li');
-    projectCardTag.innerText = projectsArray[i].techtags[j];
-    projectCardTags.appendChild(projectCardTag);
+document.addEventListener('DOMContentLoaded', () => {
+  const projectCardButtons = document.querySelectorAll('.project_card_button');
+
+  for (let i = 0; i < projectCardButtons.length; i += 1) {
+    ((index) => {
+      projectCardButtons[index].addEventListener('click', () => {
+        createPopup(index);
+        const popupWindow = document.getElementsByClassName('popup__window')[index];
+        popupWindow.style.display = 'block';
+      });
+    })(i);
   }
-
-  const projectCardButton = document.createElement('button');
-  projectCardButton.classList.add('project_card_button');
-  projectCardButton.setAttribute('type', 'button');
-  projectCardButton.innerText = 'See Project';
-  projectCardWrapper.appendChild(projectCardButton);
-
-  projectWrapper.appendChild(projectCard);
-
-  projectCardButton.addEventListener('click', () => createPopup(i));
-}
+});
 
 // Form Validation //
 
-const fullName = document.getElementById('fullname');
-const email = document.getElementById('email');
-
-fullName.removeAttribute('required');
+/* ------- SAVE INPUT VALUES TO LOCAL STORAGE -------*/
+const contactForm = document.getElementsByClassName('contact__form')[0];
+contactForm.addEventListener('submit', () => {
+  const fullNameValue = document.getElementById('fullname').value;
+  const emailValue = document.getElementById('email').value;
+  const messageValue = document.getElementById('message').value;
+  localStorage.setItem('fullName', fullNameValue);
+  localStorage.setItem('email', emailValue);
+  localStorage.setItem('message', messageValue);
+});
```

Comment:
- [ ] Seems like you are saving the values to the storage when the submit button clicked, please watch out the requirements. And please have more commits from your partner as it will help you to work equally. 
the requirements: 
![image](https://github.com/mateoburvi/Portfolio/assets/91486791/11bd0925-90e3-4183-b4c7-78da26a4f8cb)
- - - - - - 
![image](https://github.com/mateoburvi/Portfolio/assets/91486791/34ddb978-ffde-4921-b9bf-3812abc1b818)
- This is an example of saving an object to local storage. Please take attention to the JSON method(stringify, also parse methods): 
``localStorage.setItem('contactFormInput', JSON.stringify(formData));``
- [ ] In the example above the `formData` is an object. Please add all your form data to one object and save it to local storage as above.